### DISCUSSION TITLE
Management Approval Process for New Articles

### DISCUSSION THREAD

**Discussion by ajdimac** [requarks/wiki#5118] [Help / Questions]
What is the best practice for setting up a management approval process for new articles. We, as a lab, need to control and confirm that new articles are vetted before being released and published to our team.

I assume that using the group permission controls can do this. **This is for Wiki.js**

Please advise.

Thanks,

ajdimac

**Comment by MarcBollmann** [comment]
This feature isn't implemented yet you can vote for it here: https://js.wiki/feedback/p/page-approval-function
